examples/nlp/gpt/conf/gpt_dpo.yaml

```diff
@@ -114,6 +114,7 @@ model:
     data_impl: jsonl
     splits_string: null
     seq_length: ${model.encoder_seq_length}
+    pad_length_to_multiple_of: null  # If using sequence_parallel, ensure disible by tensor_model_parallel_size.
```

Comment:
Should be `divisible by tensor_model_parallel_size`

Reply:
thanks for catching; i've updated